### DISCUSSION TITLE
Add support for IDNs

### DISCUSSION THREAD

**Discussion by wpbft** [plausible/analytics#1304] [Anything Else]
Currently [IDNs](https://help.zone.eu/en/kb/using-idns-in-web-browsers-and-email-clients/) are not supported directly, and you have to use ACE strings to bypass this.

Example: the ACE-string of the domain name JÕULUD.EU is XN–JULUD-DUA.EU.

It would be nice to have IDNs displayed correctly in the Plausible dashboard.

**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.

**Comment by ukutaht** [comment]
Can you post steps to reproduce? I cannot even create a site with non-ascii characters in the domain name:

![Screenshot from 2021-09-07 15-45-03@2x](https://user-images.githubusercontent.com/3731516/132346888-3cfdc40b-11da-4246-9b11-942086cc179a.png)

**Reply by wpbft** [reply]
You're right - I have corrected my original post.

**Comment by abeldotam** [comment]
Any news on this subject? I wanted to create a new suggestion but I see this one is already open.

The IDN aren't still managed on Plausible, for instance the website [https://ingénieur.fr](https://ingénieur.fr) _(🇫🇷 ingénieur = 🇬🇧 engineer)_ will be displayed as [https://xn--ingnieur-d1a.fr](https://ingénieur.fr), which is a shame because it distorts the name, recognition or meaning of the domain name.

Another example meaning in my case, [https://èé.fr](https://èé.fr) will be displayed as [https://xn--8cac.fr](https://èé.fr)

![image](https://user-images.githubusercontent.com/5216201/179504438-3d515c4d-ca7e-40ee-9469-1109261118fb.png)

---

This kind of domain can be used by a last part of the population.
More explanation of them:

> An internationalized domain name (IDN) is an [Internet](https://en.wikipedia.org/wiki/Internet) [domain name](https://en.wikipedia.org/wiki/Domain_name) that contains at least one label displayed in [software applications](https://en.wikipedia.org/wiki/Application_software), in whole or in part, in non-latin script or [alphabet](https://en.wikipedia.org/wiki/Alphabet), such as [Arabic](https://en.wikipedia.org/wiki/Arabic), [Bengali](https://en.wikipedia.org/wiki/Bengali_language), [Chinese](https://en.wikipedia.org/wiki/Chinese_character) ([Mandarin](https://en.wikipedia.org/wiki/Mandarin_Chinese), [simplified](https://en.wikipedia.org/wiki/Simplified_Chinese_characters) or [traditional](https://en.wikipedia.org/wiki/Traditional_Chinese_characters)), [Cyrillic](https://en.wikipedia.org/wiki/Cyrillic_alphabet) (including [Bulgarian](https://en.wikipedia.org/wiki/Bulgarian_language), [Russian](https://en.wikipedia.org/wiki/Russian_language) and [Ukrainian](https://en.wikipedia.org/wiki/Ukrainian_language)), [Devanagari](https://en.wikipedia.org/wiki/Devanagari), [Greek](https://en.wikipedia.org/wiki/Greek_alphabet), [Hebrew](https://en.wikipedia.org/wiki/Hebrew_language), [Hindi](https://en.wikipedia.org/wiki/Hindi), [Tamil](https://en.wikipedia.org/wiki/Tamil_language) or [Thai](https://en.wikipedia.org/wiki/Thai_language) or in the [Latin alphabet](https://en.wikipedia.org/wiki/Latin_alphabet)-based characters with [diacritics](https://en.wikipedia.org/wiki/Diacritics) or [ligatures](https://en.wikipedia.org/wiki/Typographic_ligature), such as [French](https://en.wikipedia.org/wiki/French_language), [German](https://en.wikipedia.org/wiki/German_language), [Italian](https://en.wikipedia.org/wiki/Italian_language), [Polish](https://en.wikipedia.org/wiki/Polish_language), [Portuguese](https://en.wikipedia.org/wiki/Portuguese_language) or [Spanish](https://en.wikipedia.org/wiki/Spanish_language)

Thanks for reading this. ✌️

**Reply by metmarkosaric** [reply]
Thanks for the feedback @abeldotam! No news at the moment but it's something we want to fix in the future. There's also this thread #993. At the moment, you could also use ingenieur.fr instead of ingénieur.fr

**Comment by metmarkosaric** [comment]
**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.
